docs/package/gluon-config-mode-geo-location.rst

```diff
@@ -0,0 +1,57 @@
+gluon-config-mode-geo-location
+==============================
+
+This package allows the user to set latitude, longitude and optionally altitude
+to be advertised from within the config mode. There are two types of this
+package:
+
+It is possible to include ``gluon-config-mode-geo-location`` or
+``gluon-config-mode-geo-location-with-map`` in the ``site.mk``.
+
```

Comment:
It is possible to include **either** ``gluon-config-mode-geo-location`` **or**
``gluon-config-mode-geo-location-with-map`` in the ``site.mk``.

If you want to use `gluon-config-mode-geo-location-with-map` together with the GLUON_FEATURE `web-wizard` then you have to exclude `gluon-config-mode-geo-location`, i.e.:

```
GLUON_SITE_PACKAGES += \
  -gluon-config-mode-geo-location \
  gluon-config-mode-geo-location-with-map
```

Reply:
I Think this is redundancy in the gluon doc ... but I'll include it anyway.
applied